endaq/plot/plots.py

```diff
@@ -1031,10 +1076,11 @@ def _add_df(fig, df, line_color, units):
 def pvss_on_4cp(
         df: pd.DataFrame,
         mode: typing.Literal["srs", "pvss"] = "srs",
-        disp_units: typing.Literal["in", "mm"] = "in",
+        accel_units: str = "gravity",
+        disp_units: str = "in",
         tick_spacing: typing.Literal["fine", "medium", "coarse"] = "medium",
         include_text: bool = True,
-        size: int = 600,
+        size: int = None,
```

Comment:
Should be: `size: typing.Optional[int] = None`

There are other places where arguments specify a data type but default to `None`, many predating this PR. These should all be fixed. Not a big deal, but linting in various IDEs will complain.